src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -369,20 +369,20 @@ def _validate_vm_create_storage_profile(cmd, namespace, for_scale_set=False):
 
     elif namespace.storage_profile == StorageProfile.ManagedSpecializedOSDisk:
         required = ['os_type', 'attach_os_disk']
-        forbidden = ['os_disk_name', 'os_caching', 'storage_account',
+        forbidden = ['os_disk_name', 'os_caching', 'storage_account', 'ephemeral_os_disk'
```

Comment:
Have we confirmed with the CRP team on this?

Reply:
Yes I confirmed with Jing xu.